showcase.js

```diff
@@ -24,6 +24,6 @@ if (navigator.vibrate) {
         });
     }
 
-    (function() {
-        statements
+    (function(ev) {
+    console.log("you can try making your own theme, also this api will drain your battery! Its too much power!")
```

Comment:
This is an example of a self-executing, anonymous function, but there are no examples of named functions and meethods.

Reply:
I have fixed this in the new pull request.